SonarQube.Bootstrapper/BootstrapperSupportedVersions.cs

```diff
@@ -1,5 +1,5 @@
 ﻿//-----------------------------------------------------------------------
-// <copyright file="BoostrapperSupportedVersions.cs" company="SonarSource SA and Microsoft Corporation">
+// <copyright file="BootstrapperSupportedVersions.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
This is the format that was agreed with LCA (our legal people). I'll check whether the file name is required.